Furball.Engine/Engine/DevConsole/ConsoleDrawable.cs

```diff
@@ -12,12 +13,12 @@ public class ConsoleDrawable : UiTextBoxDrawable {
             this.OriginType = OriginType.Center;
             this.Visible    = false;
 
-            this.OnCommit                      += this.OnTextCommit;
-            FurballGame.InputManager.OnKeyDown += this.OnKeyDown;
+            this.OnCommit                  += this.OnTextCommit;
+            Keyboard.GetKeyboard().KeyDown += this.OnKeyDown;
```

Comment:
Why arent we using the InputManager here?

Reply:
idk, fixed